Lesson15/Gokhan/script.js

```diff
@@ -0,0 +1,22 @@
+// 4. Use the `toLowerCase` method to convert a string to lowercase and print it to the console.
+const stringExercise4 =
+	'FUNCTIONAL PROGRAMMING EMPHASIZES PURE FUNCTIONS AND IMMUTABLE DATA.';
+
+console.log('Exercise 4: ');
+console.log(stringExercise4.toLowerCase());
+
+// 6. Use the `lastIndexOf` method to find the last index of of "l" character in the string stringExercise5 and print it to the console.
+
+console.log('Exercise6: ');
+console.log(stringExercise5.lastIndexOf("l"));
+
+// 15. Remove extra spaces from the string stringExercise15 and print the result to the console.
+const stringExercise15 = '    Hello world    ';
+
+console.log('Exercise15: ');
+console.log(stringExercise15.trim());
+// 16. Extract the price and currency from a string below using "slice" method and print it to console.
+const planPriceString = 'Premium plan - 9.99 USD/month.'; // Should return "9.99 USD"
+
+console.log('Exercise16: ');
+console.log(planPriceString.slice(15, 23));
```

Comment:
In real life it's usually not possible to manually calculate indexes of certain values and rely on them since data is dynamic in most cases. Instead, we have to rely on certain patterns.
In this case, it would be better to find the start index of the price (e.g. you can see that there's a '- ' just before the price, let's assume it's a common pattern) so you can do something like this: `const startIndex = planPriceString.indexOf('- ')+2;`.
This operation will first find the index of '-' then add 2 to it so the `startIndex` variable will be at the start index of the price amount.

For the last index, you can search for the '/' symbol in a similar way. 
When both indexes are calculated, add them to the slice method that you have already.